app/code/Magento/MediaStorage/Model/File/Validator/Image.php

```diff
@@ -60,10 +69,8 @@ public function isValid($filePath): bool
 
         if (in_array($fileMimeType, $this->imageMimeTypes)) {
             try {
-                //phpcs:ignore Magento2.Functions.DiscouragedFunction
-                $image = imagecreatefromstring($this->file->fileGetContents($filePath));
-
-                $isValid = $image ? true : false;
+                $imagick = $this->imageFactory->create($filePath);
+                $imagick->open();
```

Comment:
```suggestion
                $image = $this->imageFactory->create($filePath);
                $image->open();
```

Reply:
It's not an issue but I believe that use of `$image` as variable name is more correct